README.md

```diff
@@ -186,6 +186,86 @@ fastify.listen(3000, function (err) {
 ```
 *NB you can find more information about Redis streams and the relevant commands [here](https://redis.io/topics/streams-intro) and [here](https://redis.io/commands#stream).*
 
+## Redis connection error
+Majority of errors are silent due to the `ioredis` silent error handling but during the plugin registration it will check that the connection with the redis instance is correctly estabilished.
+In this case you can receive an `ERR_AVVIO_PLUGIN_TIMEOUT` error if the connection cant be estabilished in the expected time frame or a dedicated error for an invalid connection.
+
+If you are using the default fastify `pluginTimeout` option and the default `maxRetriesPerRequest` redis option you will probably encounter an `ERR_AVVIO_PLUGIN_TIMEOUT` error:
+```js
+'use strict'
+
+const fastify = require('fastify')()
+const fastifyRedis = require('fastify-redis')
+
+fastify
+  .register(fastifyRedis, {
+    host: 'invalid_host',
+  })
+
+fastify.listen(3000, err => {
+  if (err) throw err
+  console.log(`server listening on ${fastify.server.address().port}`)
+})
+```
+```
+Error: ERR_AVVIO_PLUGIN_TIMEOUT: plugin did not start in time: /fastify-redis/index.js. You may have forgotten to call 'done' function or to resolve a Promise
```

Comment:
Do we need to show the full error message inside the README? I think a sentence at start is enough.

Reply:
Dont know tbh. :thinking: 
I thought could be useful to differentiate the two different errors that the user could encounter and explaining why.
If you have a strong opinion regarding this we can for sure keep only the first sentence.

Reply:
I do not have strong opinion on it. Just wondering what make you think the error should exist inside the README. If you think it will be useful, I am totally fine with it.

Reply:
I think you were right about this, no need to show error examples on README. Thanks for the tip :pray: See commit [d3064c0](https://github.com/fastify/fastify-redis/pull/98/commits/d3064c00b2bdcb916d75ba0301066983a2c3a7b3)